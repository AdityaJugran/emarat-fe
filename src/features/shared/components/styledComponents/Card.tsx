import tw from 'twin.macro';
import styled from 'styled-components';

export default styled.div(() => [
	tw`
		flex
		items-center
		bg-white
		shadow-lg
		py-5
		px-8
		h-full
		w-full
	`,
]);
