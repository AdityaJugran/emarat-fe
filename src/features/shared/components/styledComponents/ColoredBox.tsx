import styled from 'styled-components';

export const ColoredBox = styled.div`
	width: ${(props: { size?: string }) => props.size || '1em'};
	height: ${(props: { size?: string }) => props.size || '1em'};
	background: ${(props: { color?: string }) => props.color || '#fff'};
`;
