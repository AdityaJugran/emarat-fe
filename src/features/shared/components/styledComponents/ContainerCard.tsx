import tw from 'twin.macro';
import styled from 'styled-components';

export default styled.div`
	${tw`
		mx-auto
		bg-white
		shadow-lg
		p-2
		h-full
        sm:mt-2
	`}
	width: ${(props: { width?: string }) => props.width || ''};
`;
