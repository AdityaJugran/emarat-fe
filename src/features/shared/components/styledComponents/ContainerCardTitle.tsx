import tw from 'twin.macro';
import styled from 'styled-components';

export default styled.div`
	${tw`
		text-3xl
		mb-8
	`}
`;
