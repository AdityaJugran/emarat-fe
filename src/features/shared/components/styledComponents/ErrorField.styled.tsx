import styled from 'styled-components';
import tw from 'twin.macro';

export default styled.div`
	${tw`
        text-red-500
        font-semibold
        mt-1
    `}
`;
