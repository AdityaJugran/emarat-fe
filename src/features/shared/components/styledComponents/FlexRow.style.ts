import styled from 'styled-components';
import tw from 'twin.macro';

const FlexRow = styled.div`
	${tw`
        flex
        flex-wrap
        justify-between
    `}
`;

export default FlexRow;
