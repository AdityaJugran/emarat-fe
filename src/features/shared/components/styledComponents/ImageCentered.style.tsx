import styled from 'styled-components';
import tw from 'twin.macro';

export default styled.img`
	${tw`
        mx-auto
        pt-5
    `}
`;
