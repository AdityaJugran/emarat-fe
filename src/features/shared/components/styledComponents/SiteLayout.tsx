import styled from 'styled-components';

export const SiteLayout = styled.div`
	margin-left: ${(props: { collapsed: boolean }) =>
		props.collapsed ? '80px' : '200px'};
	width: ${(props: { collapsed: boolean }) =>
		props.collapsed ? 'calc(100% - 80px)' : 'calc(100% - 200px)'};
	@media (max-width: 640px) {
		margin-left: 0px;
		width: 100%;
	}
`;
