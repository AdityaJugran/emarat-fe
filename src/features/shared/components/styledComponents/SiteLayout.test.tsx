import renderer from 'react-test-renderer';
import { SiteLayout } from 'features/shared/components/styledComponents/SiteLayout';

it('SiteLayout styled', () => {
	const tree = renderer.create(<SiteLayout collapsed />).toJSON();
	expect(tree).toMatchSnapshot();

	const tree2 = renderer.create(<SiteLayout collapsed={false} />).toJSON();
	expect(tree2).toMatchSnapshot();
});
