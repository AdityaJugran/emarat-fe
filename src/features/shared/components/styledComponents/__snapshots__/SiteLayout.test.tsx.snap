// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SiteLayout styled 1`] = `
<div
  className="sc-bdfBQB fVssRp"
/>
`;

exports[`SiteLayout styled 2`] = `
<div
  className="sc-bdfBQB eFjETN"
/>
`;
